tests/serverpod_test_server/test/method_path.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:http/http.dart' as http;
+
+import 'config.dart';
+
+import 'package:test/test.dart';
+
+// These tests check that the server support both calling methods through
+// parameters and through the new path format (e.g. /simple/hello), which is
+// required by OpenAPI.
+
+void main() {
```

Comment:
Great addition of tests! ⭐ 

These can probably be used in the documentation for examples of how endpoints can be called.